src/app/frank/fd_frank_init

```diff
@@ -78,6 +78,17 @@ for((verify_idx=0;verify_idx<VERIFY_CNT;verify_idx++)); do
     || exit $?
 done
 
+# Create the network namespace(s)
+ip netns add fd-$APP-netless
+
+# Create the thread group users
+useradd -MN -s /sbin/nologin fd-$APP-monitor
+useradd -MN -s /sbin/nologin fd-$APP-verify
+useradd -MN -s /sbin/nologin fd-$APP-dedup
+useradd -MN -s /sbin/nologin fd-$APP-pack
```

Comment:
I initially thought that we'd need users to gate access to different workspaces but it turns out we don't need users for that. Since shmem is access through hugetblfs, the process can mmap the needed pages and then sandbox itself.
It won't be able to:
1- use mmap (seccomp)
2- see the different wksps since they aren't mounted in its new mountns